app/src/main/java/tech/relaycorp/gateway/background/PingRemoteServer.kt

```diff
@@ -1,55 +1,40 @@
 package tech.relaycorp.gateway.background
 
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.android.Android
+import io.ktor.client.request.head
+import io.ktor.network.selector.ActorSelectorManager
+import io.ktor.network.sockets.aSocket
 import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.withContext
 import tech.relaycorp.gateway.common.Logging.logger
 import java.io.IOException
-import java.net.ConnectException
-import java.net.HttpURLConnection
-import java.net.InetSocketAddress
-import java.net.Socket
-import java.net.URL
 import java.util.logging.Level
 import javax.inject.Inject
 
 class PingRemoteServer
 @Inject constructor() {
-    suspend fun pingAddress(address: String, port: Int) =
-        withContext(Dispatchers.IO) {
-            val socket = Socket()
-            try {
-                socket.connect(InetSocketAddress(address, port), TIMEOUT)
-                true
-            } catch (ce: ConnectException) {
-                logger.log(Level.INFO, "Could not reach $address:$port")
-                false
-            } catch (ex: Exception) {
-                logger.log(Level.INFO, "Could not reach $address:$port", ex)
-                false
-            } finally {
-                socket.close()
-            }
+
+    private val ktorClient by lazy {
+        HttpClient(Android)
```

Comment:
Can you please add the user agent? `Relaynet Private Gateway/<version> (Android)`. Or it isn't easy to get the version of the app, just `Relaynet Private Gateway (Android)`.

That'd be helpful when I'm looking at the server logs.